org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DefaultDesktop.java

```diff
@@ -563,7 +566,7 @@ public void renderHomeTab()
 	{		
 		homeTab.getChildren().clear();		
 
-		dashboardController.render(homeTab, this, true);
+		dashboardController.render(homeTab, m_WindowNo, this, true);
```

Comment:
please remove m_WindowNo instance variable, as I've pointed above, it is not needed.